packages/boxel-motion/addon/behaviors/spring.ts

```diff
@@ -56,24 +56,34 @@ export default class SpringBehavior implements Behavior {
     assert('Damping value must be greater than 0', this.options.damping > 0);
   }
 
-  toFrames(options: SpringToFramesArgument): Frame[] {
+  *getFrames(options: SpringToFramesArgument) {
     let { from, to, velocity = 0, delay = 0 } = options;
 
+    if (from === to) {
+      return [];
```

Comment:
Should we be returning `undefined` here?

Reply:
Should be resolved